openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml

```diff
@@ -704,18 +704,14 @@
 - control_key: SC-10
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - text: |
-        The customer will be responsible for terminating network connections
-        at the end of the session or after no longer than 30 minutes of
-        inactivity. A successful control response will document the technical
-        means on how this is established.
-
-        A complete control response is planned. Engineering progress can be
-        tracked via:
+        The OpenShift Oauth Server has a setting that enables the termination
+        of inactive sessions. The relevant setting is `accessTokenInactivityTimeout`
+        and it's possible to configure it under the OAuth object. [1]
```

Comment:
Check out https://github.com/ComplianceAsCode/redhat/blob/master/openshift-container-platform-4/policies/AC-Access_Control/component.yaml#L168 -- the inactivity timeout can unfortunately be configured either per-client or globally (and for the UI, there is also a separate configMap...) and the global setting doesn't propagate to the client.

I guess for the control reply your text could be OK, but for the sake of the rule, we should check both.